Update ClickHouse doc (#5900)
dbt-labs/docs.getdbt.com: website/docs/docs/core/connect-data-platform/clickhouse-setup.md

  outputs:
    <target-name>:
      type: clickhouse
      schema: [ default ] # ClickHouse database for dbt models

      # optional

>       driver: [ http ] # http or native.  If not set this will be auto-determined based on port setting

Reviewer: ```suggestion
      driver: [ http ] # http or native.  If not configured, this will be auto-determined based on the port setting
```

      host: [ localhost ]

>       port: [ 8123 ]  # If not set, defaults to 8123, 8443, 9000, 9440 depending on the secure and driver settings 

Reviewer: ```suggestion
      port: [ 8123 ]  # Defaults to 8123, 8443, 9000, 9440 depending on the secure and driver settings 
```

      user: [ default ] # User for all database operations
      password: [ <empty string> ] # Password for the user

>       cluster: [ <empty string> ] # If set, certain DDL/table operations will be executed with the `ON CLUSTER` clause using this cluster. Distributed materializations require this setting to work. See the following ClickHouse Cluster section for more details.

Reviewer: ```suggestion
      cluster: [ <empty string> ] # If configured, certain DDL/table operations will be executed with the `ON CLUSTER` clause using this cluster. Distributed materializations require this setting to work. See the following ClickHouse Cluster section for more details.
```
